languages/csharp/exercises/concept/casting/.meta/Example.cs

```diff
@@ -0,0 +1,59 @@
+
+public class LaminateMaker
+{
+    private bool securityAlert;
+    public LaminateMaker(bool securityAlert)
+    {
+        this.securityAlert = securityAlert;
+    }
+    public string GetDisplayName(TeamSupport support)
+    {
+        if (support is Staff staff)
+        {
+            string priorityPersonnelText = string.Empty;
+            if (securityAlert && staff.GetType() == typeof(Security))
+            {
+                priorityPersonnelText = " Priority Personnel";
+            }
+
+            return staff.Title + priorityPersonnelText;
+        }
+        else
+        {
+            return "Too Important for a Laminate";
+        }
+    }
+
+    public uint ConvertShirtNum(ulong shirtNum)
```

Comment:
The numbers exercise also has students converted between numeric types. See the [after.md document](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/numbers/.docs/after.md) and the [example](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/numbers/.meta/Example.cs).

I'm wondering if we could get away with not having this method at all as the student is guaranteed to have done this already.

Reply:
Removed